core-web/libs/sdk/types/src/lib/client/public.ts

```diff
@@ -0,0 +1,95 @@
+/**
+ * The parameters for the Page API.
+ * @public
+ */
+export interface PageRequestParams {
```

Comment:
We can combine the `GraphQLPageOptions` and `PageRequestParams` in one for this method